utils/perl5lib/Batch/BatchMaker.pm

```diff
@@ -812,7 +812,7 @@ sub setCESMRun()
         }
     }
 
-    $self->{'mpirun'} .= "qx( " . $executableString . " " . $mpiargstring .  " " . $defaultrunsuffix . ");";
+    $self->{'mpirun'} .= "run_cmd( " . $executableString . " " . $mpiargstring .  " " . $defaultrunsuffix . ")";
```

Comment:
Done. I am not using BatchMaker.pm. Jim will be getting rid of as much of this as possible in his PR.